scripts/linters/custom_eslint_checks/rules/protractor-practices.js

```diff
@@ -53,6 +55,19 @@ module.exports = {
     };
 
     return {
+      'VariableDeclaration[kind=const]': function(node) {
+        var constantName = node.declarations[0].id.name;
+        if ((node.declarations[0].id.type === 'Identifier') &&
+          (constantName !== constantName.toUpperCase())) {
+          context.report({
+            node: node,
+            messageId: 'constInAllCaps',
+            data: {
+              constName: constantName
+            }
+          });
+        }
```

Comment:
I would suggest moving it to a separate function.

Reply:
Done!